libethereum/DatabasePaths.cpp

```diff
@@ -0,0 +1,82 @@
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2014-2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+#include "DatabasePaths.h"
+#include "libdevcore/CommonIO.h"
+#include "libdevcore/DBFactory.h"
+#include "libethcore/Exceptions.h"
```

Comment:
Is this needed?

Reply:
Ah nice catch - it used to be since I would throw if an empty genesis hash was supplied, but I pulled that out because tests use empty hashes. I’ll remove the header.